Makefile

```diff
@@ -204,6 +204,10 @@ test-vendor: vendor
 test-sqlite: integrations.sqlite.test
 	GITEA_ROOT=${CURDIR} GITEA_CONF=integrations/sqlite.ini ./integrations.sqlite.test
 
+.PHONY: test-sqlite-%
+test-sqlite-%: integrations.sqlite.test
```

Comment:
Doesn't this will conflict with `test-sqlite-migration` ?

Reply:
You're right. Thanks.
Let me think about other ways.

Reply:
I think make will allow specific things to override wildcards. So although it technically conflicts it might not matter. I also don't think there's a specific test called Migration that we'd be conflicting with.

If not it's probably ok to change the name of the migration test infrastructure.

Reply:
Otherwise, `make test-sqlite#RepoCommitsSearch` can probably work too.
Let me give a try.

Reply:
Ah, `#` is for commenting :(

Reply:
For the record, `#` does work after proper escaping.
I think `#`, `@`,`$` and `_` would probably all work.
I will settle on `#` for now.